### PR TITLE
VACMS-9364: Remove copay hours from billing.

### LINKED ISSUE #9364: Remove deprecated field_hours_for_copay_inquiries_

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
closes #9364

## Testing done
Visual / updb / cim / Behat

## QA steps
 - [ ] As an administrator, go to admin/structure/types/manage/vamc_system_billing_insurance/fields and confirm page does not show field_hours_for_copay_inquiries_
- [ ] Confirm no system messages or page errors related to missing fields / data

### Definition of Done
- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -187,7 +187,7 @@
         "symfony/process": "^4.0",
         "symfony/routing": "^4.0",
         "traviscarden/behat-table-comparison": "~0.2",
-        "va-gov/content-build": "^0.0.3032",
+        "va-gov/content-build": "^0.0.3033",
```
Comment (JayDarnell) [inline composer.json:190]: @ethanteague QA passes and these changes all look good with the exception of this line. This seems like something that may have been accidentally picked up.

**Review by JayDarnell — CHANGES_REQUESTED** [review]
Marking as needs possible changes based on the changes to composer.json

File: composer.json (hunk 1 above)
Comment (ethanteague) [reply composer.json:190]: Yep. Need it to grab https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9362

**Review by JayDarnell — APPROVED** [review]
QA passes and code changes look good.

**Pull request merged**